content/en/_index.md

```diff
@@ -9,25 +9,25 @@ type: docs
 
 Interlisp was a pioneering software development environment from the 1970s at Xerox PARC targeted primarily at the AI software development community. A group of its developers and users have gotten together to revive Interlisp -- as a language, environment, spirit -- in the Medley Interlisp Project.
 
-# [Project Overview](project)
+# [What is the Medley Interlisp Project?](project)
 What we are doing and why.
 
-# [About the software](about)
+# [What does Interlisp have to do with current systems?](about)
 What are the main features that make it interesting to revive?
 How might we apply the lessons from Interlisp to current systems?
 
-# [Using Medley today](using)
+# [How do I try using Interlisp?](using)
```

Comment:
"There is no try, there is only do."
I think we're calling the system we're maintaining Medley, the project the Medley Interlisp Project...
But "How can I get medley?" or "How do I run Medley?" 
I want the topic to cover not only how to get it, but how to use it once you have. "How do I try it?" (OK, there is a "try" sometimes).